Formulario prueba

Valentina2103 opened the pull request.

Valentina2103 committed:
formulario prueba

redclazz2 reviewed (changes requested):
Dividir la función de Validaciones en funciones individuales, de forma que no se tenga que procesar la lista de nombres prohibidos si cambia p ej la apuesta

redclazz2 committed:
Added Cookie Support

Added cookies to save data related to room creation. (This change comes with two npm packages).
Removed wrong implementation of JQUERY import on multiple files.
Fixed some errors on form validations

redclazz2 reviewed (approved):
LGTM

redclazz2 commented:
Por hacer: 
1. Cuando el select de apuesta es si, el valor de la apuesta no puede ser menor a 1.
2. Cambiar el algoritmo de la verificación de las palabras obsenas en el nombre por uno más eficiente.
